staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5871,7 +5871,9 @@ type NodeDaemonEndpoints struct {
 	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty" protobuf:"bytes,1,opt,name=kubeletEndpoint"`
 }
 
-// NodeRuntimeHandlerFeatures is a set of runtime features.
+// NodeRuntimeHandlerFeatures is a set of features implemented by the runtime handler.
+// The features contained in the NodeRuntimeHandlerFeatures should depend on the runtime handlers,
+// (i.e. dependent to the information exposed by the OCI runtime-spec's features).
```

Comment:
Same comment as above.

Reply:
Fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66